.github/workflows/dockerimage.yml

```diff
@@ -43,3 +44,11 @@ jobs:
     - name: Build the Docker image
       run: |
         docker build . --build-arg PYTHON_VERSION=${{ matrix.python_version }} --build-arg BUILD_TYPE=${{ matrix.build_type }} --file Dockerfile --tag "${HUB_TAG}"
+
+    - name: Login & Push to Docker Hub
```

Comment:
Personally, this step should only be triggered by master push and release.

Reply:
That makes sense. We can keep the build step for testing purposes, and we'll be aware of the failure of docker build. Fixed in https://github.com/optuna/optuna/pull/1295/commits/dd16c2aeba3cfe1f15b2cadc6cbcec9bf1c5229c.
Thank you for your suggestion.